support/Pipelines/Bbh/InitialData.yaml

```diff
@@ -5,19 +5,18 @@ Executable: SolveXcts
 ExpectedOutput:
   - BbhReductions.h5
   - BbhVolume*.h5
-{% if evolve %}
+  - BbhSurfaces.h5
 Next:
-  Run: spectre.Pipelines.Bbh.Inspiral:start_inspiral
+  Run: spectre.Pipelines.Bbh.InitialData:postprocess_id
   With:
     id_input_file_path: __file__
     id_run_dir: ./
-    pipeline_dir: {{ pipeline_dir }}
-    continue_with_ringdown: True
+    pipeline_dir: {{ pipeline_dir | default("None") }}
+    evolve: {{ evolve }}
```

Comment:
Does this now not have the ability to choose whether to continue with ringdown? I see in `InitialData.py` that `continue_with_ringdown` is always True.

Reply:
If you run with `--evolve` it will also just try to continue with ringdown (which isn't working yet). That's the same behavior as before. I could disable that since it's not working yet?

The `start-inspiral` command still has the option whether or not to continue with ringdown.